include/dml.h

```diff
@@ -52,7 +52,7 @@ typedef struct {
 typedef uint32_t DML_SiteRank32;
 typedef int64_t DML_SiteRank;
 
-typedef int64_t DML_Index;
+typedef n_uint64_t DML_Index;
```

Comment:
This changes the interface.  Is it necessary?

Reply:
I tried to make all indices unsigned to prevent rollover related issues.